pontoon/sync/templates/sync/log_list.html

```diff
@@ -33,6 +33,9 @@
             <th class="latest-status">Latest Status<i class="fas"></i></th>
             <th class="latest-start relative-time">Latest Start<i class="fas"></i></th>
             <th class="duration" data-sort="">Duration<i class="fas"></i></th>
+            {% if settings.MANUAL_SYNC and is_manager %}
```

Comment:
A general observation, since this is from the previous PR: `request` is available in the templates, so passing `is_manager` is not needed.

Reply:
But then you would need to repeat the condition `request.user.has_perm("base.can_manage_project")` 3 times in this template (or enclose everything in a `with`), which doesn't seem more readable? Is there a better way to do that?

Reply:
You could [declare a variable in the template](https://jinja.palletsprojects.com/en/stable/templates/#assignments):
```jinja
{% set is_manager = request.user.has_perm("base.can_manage_project") %}
```